CONTRIBUTING.md

```diff
@@ -304,9 +407,26 @@ Follow the coding guidelines here - [C# Coding Conventions (C# Programming Guide
      }
      ```
 
+### Backend Unit Testing
```

Comment:
New section, figured we should add this to match with the content in the Frontend section

Reply:
Yes, it looks good!